src/main/resources/com/parasoft/findings/jenkins/coverage/cobertura.xsl

```diff
@@ -42,6 +76,13 @@
         <xsl:element name="packages">
 <!--             Group by the parent path of uri-->
             <xsl:for-each-group select="/Coverage/Locations/Loc" group-by="substring-before(@uri, tokenize(@uri, '/')[last()])">
+                <xsl:variable name="rawUri" select="@uri"/>
```

Comment:
1. We can add a variable named getuUriWithoutFilePrefix.
```
<xsl:variable name="uriWithoutFilePrefix">
     <xsl:call-template name="getuUriWithoutFilePrefix">
        <xsl:with-param name="uri" select="$rawUri"/>
    </xsl:call-template>
</xsl:variable>
```
2. Add a templat named `getuUriWithoutFilePrefix`, and process uri in this template.
3. Use uriWithFileInitialize where previously used uri